src/sage/combinat/permutation.py

```diff
@@ -7092,7 +7092,17 @@ def _element_constructor_(self, x, check=True):
             [1, 4, 5, 2, 3, 6]
             sage: Permutations(6)(x)            # known bug
             [1, 4, 5, 2, 3, 6]
+
+        Ensure that :issue:`37284` is fixed::
+
+            sage: PG = PermutationGroup([[(1,2,3),(5,6)],[(7,8)]])
+            sage: P5 = Permutations(8)
+            sage: p = PG.an_element()
+            sage: P5(p).parent()
+            Standard permutations of 8
```

Comment:
```suggestion
            sage: P8 = Permutations(8)
            sage: p = PG.an_element()
            sage: P8(p).parent()
            Standard permutations of 8
```
Since the group is larger now. It would be good to display the output as well.